docs/api/browser-window.md

```diff
@@ -61,6 +61,7 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
   key is pressed.
 * `enable-larger-than-screen` Boolean - Enable the window to be resized larger
   than screen.
+* `background-color` Integer - Window's background color.
```

Comment:
Using raw HEX value of ARGB is quite inconvenient, I think ideal type should be String accepting color values used in CSS, like `#FFF`, `rgb(255, 255, 255)`, `black` and `white`.

However support full CSS format value is not easy work since `blink` doesn't seem to expose related functions, we can probably just support the `#FFF` style color value, which is [easy to parse](https://code.google.com/p/chromium/codesearch#chromium/src/third_party/WebKit/Source/platform/graphics/Color.cpp&sq=package:chromium&type=cs&l=120) and everyone is familiar with it.